sql/core/src/main/java/org/apache/spark/sql/streaming/Trigger.java

```diff
@@ -20,6 +20,7 @@
 import java.util.concurrent.TimeUnit;
 
 import org.apache.spark.annotation.Evolving;
+import org.apache.spark.sql.execution.streaming.AvailableNowTrigger$;
```

Comment:
uber nit (also existing): isn't the import order messed up? Shouldn't scala be it's own group. the it should be 
```java
import org.apache.spark.annotation.Evolving;
import org.apache.spark.sql.execution.streaming.AvailableNowTrigger$;
import org.apache.spark.sql.execution.streaming.ContinuousTrigger;import 
import org.apache.spark.sql.execution.streaming.OneTimeTrigger$;
import org.apache.spark.sql.execution.streaming.ProcessingTimeTrigger;

```